app/models/quote.rb

```diff
@@ -1,3 +1,10 @@
 class Quote < ApplicationRecord
   belongs_to :prior_authority_application
+  has_one :document, lambda {
```

Comment:
We'll need to update `QuotePayloadBuilder::PriorAuthority::SubmitToAppStore` to add this new stuff to the payload - we're trying to keep this up to date as we go.

Reply:
added